scripts/restartdaq

```diff
@@ -124,7 +124,13 @@ else
     PROCMGR="/reg/g/pcds/dist/pds/$HUTCH/current/tools/procmgr/procmgr"
 fi
 
-IS_DAQ_HOST=$(netconfig search "$AIMHOST"-$DAQNETWORK --brief | grep -c $DAQNETWORK)
+# If UED machine; bypass drp node requirement, else; check for drp node
+if [ "$HUTCH" == "ued" ]; then
+    IS_DAQ_HOST=1
```

Comment:
Small question: are there hutch computers other than ued-daq where this could cause an issue?

There's a later (a few lines later) check in the script for IS_DAQ_HOST with the intention of giving suggestions on other hosts to try if you're not on a host that can run the daq. I think it'd be best if any such other hosts would recommend ued-daq to you.

Reply:
I believe ued-daq is currently the only computer where this may cause an issue.

There are other machines like ued-monitor, but you would not be able to run the daq from that machine.
Do you think it would still be beneficial to have this sort of warning you suggested in case someone on a non-daq machine tries to run `restartdaq`?

Reply:
It's probably not the most important thing, but I wonder what you expect/prefer to happen if someone runs this script on ued-monitor or one of the ioc hosts. Just food for thought, I'm not rejecting this PR or anything.

Reply:
I will add these features in a new pull request